tests/test_jobs.py

```diff
@@ -46,3 +58,13 @@ def test_schedule(self):
         sleep(5)
         s.stop(j)
         self.assertGreater(j.called, 2)
+
+    def test_schedule_log(self):
+        s = JobScheduler()
+        job_exception = TestJobRaiseException()
+        with self.assertLogs() as cm:
+            s.schedule('1s', '1s', job_exception)
+            sleep(10.1)
+            log = str(cm.output)
+            self.assertIn('BaseException', log)
+            self.assertIn('Error while running job', log)
```

Comment:
add another test please. where the job takes `10 seconds` to finish but it has a frequency of `5 seconds`
test how much time did it take it to run? 

are we running it again also when he is currently running?

thank you

Reply:
I'll add the tests, I had kept the time low for the reason is we are using `sleep` to test the jobs, i had tried `freezegun` but it is failing here so it is consuming a lot of time just for a single test to run using `sleep`

Reply:
let's resolve this on Thursday with the splash screen

Reply:
lets remove this test